src/main/content/index.html

```diff
@@ -9,116 +9,117 @@
 seo-title: Kabanero
 ---
 {% assign t = site.data[site.active_lang] %}
-<div class="container-fluid">
-    <div class="row gray-background py-5 px-5">
-        <div class="col-xl-8">
+{% asset thirdparty/carbon/carbon-components.min.css %}
+{% asset thirdparty/carbon/carbon-components.min.js %} 
+
+<div class="bx--grid--full-width">
+    <div class="bx--row gray-background py-5 px-5 ">
+        <div class="bx--col-sm-4 bx--col-md-8 bx--col-lg-8 ">
             <h1>{{t.index.welcome-kabanero}}</h1>
             <h3 id="welcome-text">{{t.index.welcome-text}}</h3>
             <a target="_blank" href="https://kabanero.io">
-                <button role="button" id="learn-more-kabanero-button" class="white-blue-hover">{{t.index.learn-more-kabanero-button}}</button>
```

Comment:
did you mean to remove the `id` here, looks like there's some scss that still uses it